tests/data/metadata/resourcemetadata.xml

```diff
@@ -135,11 +130,6 @@
       </rdf:Description>
     </dc:creator>
     <dc:subject>key 1</dc:subject>
-    <dc:source>
```

Comment:
There are also references to source in https://github.com/hydroshare/hsmodels/blob/main/tests/data/metadata/resourcemetadata_with_point_coverage.xml#L16 (and another one lower down).